javascript-ajax/users.js

```diff
@@ -0,0 +1,18 @@
+function getUserName(name) {
+  const user = document.querySelector('#user-list');
+  const xhr = new XMLHttpRequest();
+  xhr.open('GET', 'https://jsonplaceholder.typicode.com/users');
+  xhr.responseType = 'json';
+  xhr.addEventListener('load', function () {
+    console.log(xhr.status);
+    console.log(xhr.response);
+    const json = xhr.response;
+    for (let i = 0; i < json.length; i++) {
+      const li = document.createElement('li');
+      li.textContent = name;
+      li.append = user;
+    }
+  });
+  xhr.send();
```

Comment:
@annaaksenov Please verify that the users list is actually rendered on the page (see https://developer.mozilla.org/en-US/docs/Web/API/Element/append).